app/views/provider/admin/keys/edit.html.erb

```diff
@@ -1,14 +1,19 @@
-<h2>Set Custom User Key</h2>
+<div class="pf-c-card__title">
+  &nbsp;
+</div>
+<div class="pf-c-card__body">
+  <p id="invalid-key-warning" style="display:none;">&nbsp;</p>
 
-<p id="invalid-key-warning" style="display:none;">&nbsp;</p>
+  <%= semantic_form_for @cinstance, url: provider_admin_application_key_path(@cinstance, @cinstance.user_key),
+                                    remote: true,
+                                    html: { class: 'pf-c-form pf-m-limit-width' }  do |form| %>
+    <%= form.input :user_key, as: :patternfly_input, required: true, input_html: { size: 100, value: nil, required: true } %>
```

Comment:
`required: true` appears twice here, is that correct?

Reply:
No, I'll fix it